### PR TITLE
Add Sandcastle for glTF PBR extensions

### PR DESCRIPTION
**Pull request by jjhembd**
# Description

Adds a Sandcastle to demonstrate the new glTF PBR extensions.

- https://github.com/CesiumGS/cesium/pull/11970
- https://github.com/CesiumGS/cesium/pull/11988
- https://github.com/CesiumGS/cesium/pull/12006

## Testing plan

Load the [glTF PBR Extensions Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html?src=glTF%20PBR%20Extensions.html) and verify the different models load and render, and try the two lighting options.

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [ ] I have updated `CHANGES.md` with a short summary of my change
~- [ ] I have added or updated unit tests to ensure consistent code coverage~
~- [ ] I have update the inline documentation, and included code examples where relevant~
- [x] I have performed a self-review of my code

### CODE REVIEW

File: Apps/Sandcastle/gallery/glTF PBR Extensions.html [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use Viewer to start building new applications or easily embed Cesium into existing applications."
+    />
+    <meta name="cesium-sandcastle-labels" content="Showcases" />
+    <title>Cesium Demo</title>
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/glTF PBR Extensions.html:15]: Please update the `title` and `description` tags.

File: Apps/Sandcastle/gallery/glTF PBR Extensions.html [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+        "use strict";
+        //Sandcastle_Begin
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          animation: false,
+          timeline: false,
+          useBrowserRecommendedResolution: false,
+        });
+
+        const { scene, camera, clock } = viewer;
+        const {
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/glTF PBR Extensions.html:43]: This is pretty atypical when compared with how we handle imports in other Sandcastle examples. I know it's a bit more verbose, but more consistency can we use `Cesium.Cartesian3`, `Cesium.DirectionalLight`, etc?

File: Apps/Sandcastle/gallery/glTF PBR Extensions.html [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+              imageBasedLighting.specularEnvironmentMaps = undefined;
+            },
+          },
+        ];
+        Sandcastle.addToolbarMenu(lightingOptions, "lightingToolbar");
+
+        let model;
+        const modelOptions = [
+          {
+            text: "Specular Test",
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/glTF PBR Extensions.html:161]: Could we start with the Clear Coat Wicker model? It matches the thumbnail, and also provides a more "real life" use case as compared with the test models.

**Review by ggetz — COMMENTED** [review]
Thanks @jjhembd! Do we still need to maintain the dev sandcastle examples now that we have this example?

File: Apps/Sandcastle/gallery/glTF PBR Extensions.html (hunk 1 above)
Comment (jjhembd) [reply Apps/Sandcastle/gallery/glTF PBR Extensions.html:15]: I updated the description. I think the title is auto-replaced elsewhere. All the other Sandcastles have "Cesium Demo" here.

File: Apps/Sandcastle/gallery/glTF PBR Extensions.html [hunk 4]
```diff
@@ -9,9 +9,9 @@
     />
     <meta
       name="description"
-      content="Use Viewer to start building new applications or easily embed Cesium into existing applications."
+      content="Use the glTF extensions to physically-based rendering."
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/glTF PBR Extensions.html:12]:
```suggestion
      content="Use glTF extensions for physically-based rendering."
```

?

**Pull request merged**